building/trusted_applications.rst

```diff
@@ -299,3 +305,66 @@ sequence:
             return TEE_ERROR_NOT_SUPPORTED;
         }
     }
+
+.. _TASign:
+
+Signing of TAs
```

Comment:
It should be clearly stated that this applies to "REE filesystem TAs" only as described at https://optee.readthedocs.io/en/latest/architecture/trusted_applications.html#ree-filesystem-ta.

Reply:
Stated and rebased to master.